docs/guides/node-operator-manual.md

```diff
@@ -290,7 +304,7 @@ Make sure Python with pip is installed and then run:
 
 ```sh
 pip install lido-cli
-lido-cli --rpc http://1.2.3.4:8545 validate_network_keys
+lido-cli --rpc http://1.2.3.4:8545 validate_network_keys --details
```

Comment:
Just curious, what the --details do?

Reply:
Displays a number of keys for each operator

![telegram-cloud-photo-size-2-5382294424136037851-y](https://user-images.githubusercontent.com/1422379/135455027-649bb7e3-9b32-40f1-bd02-e393b4ef193c.jpg)